src/lib.rs

```diff
@@ -8,6 +8,161 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+//! a simple, humane, typed Rust interface to [LMDB](http://www.lmdb.tech/doc/)
+//!
+//! It aims to achieve the following:
+//!
+//! - Avoid LMDB's sharp edges (e.g., obscure error codes for common situations).
+//! - Report errors via [failure](../failure/index.html).
+//! - Correctly restrict access to one handle per process via a [Manager](struct.Manager.html).
+//! - Use Rust's type system to make single-typed key stores (including LMDB's own integer-keyed stores) safe and ergonomic.
+//! - Encode and decode values via [bincode](../bincode/index.html)/[serde](../serde/index.html)
+//!   and type tags, achieving platform-independent storage and input/output flexibility.
+//!
+//! It exposes these primary abstractions:
+//!
+//! - [Manager](struct.Manager.html): a singleton that controls access to LMDB environments
+//! - [Rkv](struct.Rkv.html): an LMDB environment, which contains a set of key/value databases
+//! - [Store](struct.Store.html): an LMDB database, which contains a set of key/value pairs
+//!
+//! Keys can be anything that implements `AsRef<[u8]>` or integers (when accessing an [IntegerStore](struct.IntegerStore.html)).
+//! Values can be any of the types defined by the [Value](value/enum.Value.html) enum, including:
+//!
+//! - booleans (`Value::Bool`)
+//! - integers (`Value::I64`, `Value::U64`)
+//! - floats (`Value::F64`)
+//! - strings (`Value::Str`)
+//! - blobs (`Value::Blob`)
```

Comment:
Missing `Json, Instant, and UUID`?

Reply:
My intention was to avoid displaying an exhaustive list and mention only a small set of common types. In 630743a, I've added this statement to be more explicit about that:

```rust
//! See [Value](value/enum.Value.html) for the complete list of supported types.
```